### PR TITLE
New Cal CSVs plus P18 cruise-specific Cal CSVs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes new cal csvs after vendor refurb, plus cruise-specific ADCP & METLGR cal csvs specific to Pioneer 18 cruise.
For cruise-specific csvs  - csv date is cruise start date.
Refer to Cal CSV procedure 3408-00201 for info on cruise-specific ADCP & METLGR cal coefficients.
I should also note that for the ADCP 24542 cal csv I just re-used the existing csv from P17 to reflect P18 cruise start date - this is the ADCP on the PM that was not deployed during P17 cruise (PMCI-16).

### CODE REVIEW

File: calibration/ADCPSL/CGINS-ADCPSL-21393__20220408.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21393,CC_scale_factor1,0.45,constant; CP04OSPM-00016
+21393,CC_scale_factor2,0.45,constant
+21393,CC_scale_factor3,0.45,constant
+21393,CC_scale_factor4,0.45,constant
+21393,CC_bin_size,800.0,in cm
+21393,CC_dist_first_bin,1695.0,in cm
+21393,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (asmith-whoi) [inline calibration/ADCPSL/CGINS-ADCPSL-21393__20220408.csv:8]: OK

File: calibration/ADCPSL/CGINS-ADCPSL-24664__20220408.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24664,CC_scale_factor1,0.45,constant; CP02PMUO-00019
+24664,CC_scale_factor2,0.45,constant
+24664,CC_scale_factor3,0.45,constant
+24664,CC_scale_factor4,0.45,constant
+24664,CC_bin_size,800,in cm
+24664,CC_dist_first_bin,1695,in cm
+24664,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (asmith-whoi) [inline calibration/ADCPSL/CGINS-ADCPSL-24664__20220408.csv:8]: OK
I don't think it matters, but this one has an orientation of 1, while most others have 1.0

File: calibration/ADCPTG/CGINS-ADCPTG-20495__20220408.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20495,CC_scale_factor1,0.45,constant; CP02PMCO-00017
+20495,CC_scale_factor2,0.45,constant
+20495,CC_scale_factor3,0.45,constant
+20495,CC_scale_factor4,0.45,constant
+20495,CC_bin_size,400.0,in cm
+20495,CC_dist_first_bin,901.0,in cm
+20495,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (asmith-whoi) [inline calibration/ADCPTG/CGINS-ADCPTG-20495__20220408.csv:8]: OK

File: calibration/ADCPTG/CGINS-ADCPTG-20496__20220408.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20496,CC_scale_factor1,0.45,constant; CP02PMUI-00017
+20496,CC_scale_factor2,0.45,constant
+20496,CC_scale_factor3,0.45,constant
+20496,CC_scale_factor4,0.45,constant
+20496,CC_bin_size,400.0,in cm
+20496,CC_dist_first_bin,901.0,in cm
+20496,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (asmith-whoi) [inline calibration/ADCPTG/CGINS-ADCPTG-20496__20220408.csv:8]: OK

File: calibration/METBKA/CGINS-METLGR-00017__20220408.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR017,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP01CNSM-00016
+LGR017,CC_height_of_air_humidity_measurement_m,4.255,
+LGR017,CC_height_of_air_temperature_measurement_m,4.255,
+LGR017,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR017,CC_jcool,1,Constant
+LGR017,CC_jwarm,1,Constant
+LGR017,CC_zinvpbl,600,Constant
+LGR017,CC_use_velpt,1,
```
Comment (reedan88) [inline calibration/METBKA/CGINS-METLGR-00017__20220408.csv:9]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00037__20220408.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR037,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP04OSSM-00015
+LGR037,CC_height_of_air_humidity_measurement_m,4.255,
+LGR037,CC_height_of_air_temperature_measurement_m,4.255,
+LGR037,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR037,CC_jcool,1,Constant
+LGR037,CC_jwarm,1,Constant
+LGR037,CC_zinvpbl,600,Constant
+LGR037,CC_use_velpt,1,
```
Comment (reedan88) [inline calibration/METBKA/CGINS-METLGR-00037__20220408.csv:9]: Looks good

File: calibration/METBKA/CGINS-METLGR-00008__20220408.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR008,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP03ISSM-00015
+LGR008,CC_height_of_air_humidity_measurement_m,4.255,
+LGR008,CC_height_of_air_temperature_measurement_m,4.255,
+LGR008,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR008,CC_jcool,1,Constant
+LGR008,CC_jwarm,1,Constant
+LGR008,CC_zinvpbl,600,Constant
+LGR008,CC_use_velpt,1,
```
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00008__20220408.csv:9]: OK

File: calibration/METBKA/CGINS-METLGR-00010__20220408.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR010,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP01CNSM-00016
+LGR010,CC_height_of_air_humidity_measurement_m,4.255,
+LGR010,CC_height_of_air_temperature_measurement_m,4.255,
+LGR010,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR010,CC_jcool,1,Constant
+LGR010,CC_jwarm,1,Constant
+LGR010,CC_zinvpbl,600,Constant
+LGR010,CC_use_velpt,1,
```
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00010__20220408.csv:9]: OK

File: calibration/METBKA/CGINS-METLGR-00017__20220408.csv (hunk 5 above)
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00017__20220408.csv:9]: OK

File: calibration/METBKA/CGINS-METLGR-00037__20220408.csv (hunk 6 above)
Comment (asmith-whoi) [inline calibration/METBKA/CGINS-METLGR-00037__20220408.csv:9]: OK

File: calibration/PCO2WB/CGINS-PCO2WB-C0071__20220303.csv [hunk 9]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0071,CC_cala,-0.0280, source_file PCO2W-B_SAMI-CO2_SN_C0071_Calibration_Files_2022-03-03.zip > C0071_02_CalCert.pdf
+C0071,CC_calb,1.0086,
+C0071,CC_calc,-2.0999,
+C0071,CC_calt,9.7447,
+C0071,CC_ea434,19706,constant
+C0071,CC_ea620,34,constant
+C0071,CC_eb434,3073,constant
+C0071,CC_eb620,44327,constant
```
Comment (reedan88) [inline calibration/PCO2WB/CGINS-PCO2WB-C0071__20220303.csv:9]: Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0096__20220303.csv [hunk 10]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0096,CC_cala,-0.0028,source_file PCO2W-B_SAMI-CO2_SN_C0096_Calibration_Files_2022-03-03.zip > C0096_02_CalCert.pdf
+C0096,CC_calb,0.8887,
+C0096,CC_calc,-1.9691,
+C0096,CC_calt,10.1876,
+C0096,CC_ea434,19706,Constant
+C0096,CC_ea620,34,Constant
+C0096,CC_eb434,3073,Constant
+C0096,CC_eb620,44327,Constant
```
Comment (reedan88) [inline calibration/PCO2WB/CGINS-PCO2WB-C0096__20220303.csv:9]: Looks good.

File: calibration/PHSENF/CGINS-PHSENF-P0181__20220305.csv [hunk 11]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0181,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0181_Validation_Files_2022-03-05.zip > SAMI_P0181_040322t_ValCert.pdf
+P0181,CC_ea578,94.1,
+P0181,CC_eb434,2284.0,
+P0181,CC_eb578,38676.0,
+P0181,CC_ind_off,0,
+P0181,CC_ind_slp,1,
+P0181,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSENF/CGINS-PHSENF-P0181__20220305.csv:8]: Looks good.

File: calibration/PHSENF/CGINS-PHSENF-P0182__20220305.csv [hunk 12]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0182,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0182_Validation_Files_2022-03-05.zip > SAMI_P0182_040322t_ValCert.pdf
+P0182,CC_ea578,94.1,
+P0182,CC_eb434,2284.0,
+P0182,CC_eb578,38676.0,
+P0182,CC_ind_off,0,
+P0182,CC_ind_slp,1,
+P0182,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSENF/CGINS-PHSENF-P0182__20220305.csv:8]: Looks good.

File: calibration/PHSENF/CGINS-PHSENF-P0183__20220305.csv [hunk 13]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0183,CC_ea434,17372.0,source_file PHSEN-F_SAMI-pH_SN_P0183_Validation_Files_2022-03-05.zip > SAMI_P0183_040322t_ValCert.pdf
+P0183,CC_ea578,94.1,
+P0183,CC_eb434,2284.0,
+P0183,CC_eb578,38676.0,
+P0183,CC_ind_off,0,
+P0183,CC_ind_slp,1,
+P0183,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSENF/CGINS-PHSENF-P0183__20220305.csv:8]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-21393__20220408.csv (hunk 1 above)
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-21393__20220408.csv:8]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-24664__20220408.csv (hunk 2 above)
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-24664__20220408.csv:8]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-20495__20220408.csv (hunk 3 above)
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-20495__20220408.csv:8]: Looks good.

File: calibration/ADCPTG/CGINS-ADCPTG-20496__20220408.csv (hunk 4 above)
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-20496__20220408.csv:8]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0071__20220303.csv (hunk 9 above)
Comment (asmith-whoi) [inline calibration/PCO2WB/CGINS-PCO2WB-C0071__20220303.csv:9]: OK

File: calibration/PCO2WB/CGINS-PCO2WB-C0096__20220303.csv (hunk 10 above)
Comment (asmith-whoi) [inline calibration/PCO2WB/CGINS-PCO2WB-C0096__20220303.csv:9]: OK

File: calibration/PHSENF/CGINS-PHSENF-P0181__20220305.csv (hunk 11 above)
Comment (asmith-whoi) [inline calibration/PHSENF/CGINS-PHSENF-P0181__20220305.csv:8]: OK

File: calibration/PHSENF/CGINS-PHSENF-P0182__20220305.csv (hunk 12 above)
Comment (asmith-whoi) [inline calibration/PHSENF/CGINS-PHSENF-P0182__20220305.csv:8]: OK

File: calibration/PHSENF/CGINS-PHSENF-P0183__20220305.csv (hunk 13 above)
Comment (asmith-whoi) [inline calibration/PHSENF/CGINS-PHSENF-P0183__20220305.csv:8]: OK

**Review by asmith-whoi — APPROVED** [review]
All look good

**Pull request merged**